test/functional/vimscript/server_spec.lua

```diff
@@ -152,3 +154,58 @@ describe('startup --listen', function()
     eq(addr, meths.get_vvar('servername'))
   end)
 end)
+
+describe('#client (child nvim)', function()
+  before_each(clear)
+
+  it('creates v:parent channel from $NVIM', function()
+    local nrchans = #meths.list_chans()
+    exec_lua([[return vim.fn.jobstart({ vim.v.progpath }, vim.empty_dict())]])
```

Comment:
I thought we had a convenience in the bridge where an _empty_ `{}` was acceptable for a Dict param. Is it intentional / desired that `vim.empty_dict()` for these params? @bfredl 

```
:lua lua =vim.fn.jobstart({ vim.v.progpath }, {})

E5108: Error executing lua Vim:E475: Invalid argument: expected dictionary
stack traceback:                                                                                                                     
        [C]: in function 'jobstart'
        [string ":lua"]:1: in main chunk
```